backend/src/models/api/LevelResetRequest.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express';
+
+import { ChatMessage } from '@src/models/chatMessage';
+import { LEVEL_NAMES } from '@src/models/level';
+
+export type LevelResetRequest = Request<
+	never,
+	{
+		resultingChatInfoMessage?: ChatMessage;
```

Comment:
Yes this is fine as an object. Maybe as this is the response, we can name it simply "chatInfoMessage"?